core/templates/dev/head/components/HintAndSolutionButtonsDirective.ts

```diff
@@ -21,8 +21,10 @@ oppia.directive('hintAndSolutionButtons', [
     return {
       restrict: 'E',
       scope: {},
+      bindToController: true,
```

Comment:
Shouldn't this be {}? I think it's mandated -- https://angular.io/guide/upgrade#using-component-directives

Reply:
Oh yes, my bad. I was following [this guide](https://toddmotto.com/no-scope-soup-bind-to-controller-angularjs/) as well and used both true & {} whenever as needed. I will make the updates.

Reply:
Done.

Reply:
Ah, even in the guide you linked, it looks like making bindToController into an object and moving scope properties across to it is the preferred method, so yeah I think the refactor makes sense.